backend/entities/Project.cs

```diff
@@ -176,6 +193,11 @@ public class Project : BaseEntity
         /// get/set - When the project was disposed.
         /// </summary>
         public DateTime? DisposedOn { get; set; }
+
+        /// <summary>
+        /// get/set - When the project was disposed.
+        /// </summary>
+        public DateTime? OfferAcceptedOn { get; set; }
```

Comment:
Where you planning to move this to the agency response table later?

Reply:
Same here, wasn't planning on adding this if there is only one per project.